### ISSUE TITLE
Перестают открываться некоторые сайты

### ISSUE THREAD

**Issue by PaWill68** [remittor/zapret-openwrt#35]
С конфигурацией по умолчанию ютуб, рутрэкер и прочее заблокированное открывается, но становятся недоступны некоторые обычные сайты, например https://forum.ixbt.com/

**Comment by RedGrandis** [comment]
Да, интересно почему. Конечно, можно исправить ситуацию, добавив не открывающиеся домены в /opt/zapret/ipset/zapret-hosts-user-exclude.txt

**Comment by DeduskaPihto** [comment]
Мне помогла настройка MODE_FILTER переключил в autohostlist и все открывается.

**Comment by serge73** [comment]
autohostlist, как бы помогает, но туда залетают сайты, которые должны через запрет ходить. Лучше руками в exclude запихивать или комбинировать использования авто и руками.

**Comment by Nosikmov** [comment]
Ну это же по факту не решение. Конфиг корявый получается раз он пакеты для всех сайтов портит даже не кто не в хостлисте

**Comment by skroychan** [comment]
Столкнулась с такой же проблемой, после долгого ковыряния в [документации](https://github.com/bol-van/zapret/tree/master?tab=readme-ov-file#%D1%84%D0%B8%D0%BB%D1%8C%D1%82%D1%80%D0%B0%D1%86%D0%B8%D1%8F-%D0%BF%D0%BE-%D0%B8%D0%BC%D0%B5%D0%BD%D0%B0%D0%BC-%D0%B4%D0%BE%D0%BC%D0%B5%D0%BD%D0%BE%D0%B2) нашла, что когда `zapret-hosts-user.txt` пустой (а по умолчанию он тут пустой), обходятся вообще все сайты не указанные в exclude. Можно либо поправить конфиг вручную прописав хостлисты вместо `<HOSTLIST>` или удалив эти правила совсем, либо добавить в этот файл хоть что-нибудь
